File: java/app/src/main/java/io/adrop/ads/example/model/Post.java
package io.adrop.ads.example.model;

import java.util.Random;
import java.util.UUID;

public class Post {
    public String id = UUID.randomUUID().toString();
    public String icon = "https://storage.adrop.io/public/ic_openrhapsody.png";
    public String title = "Adrop";
    public String subTitle = "OpenRhapsody";
    public String image = "https://storage.adrop.io/public/openrhapsody_title.png";
    public String content = "Adrop Post Text";
    public int like = new Random().nextInt(100);
    public int comment = new Random().nextInt(100);
}

File: kotlin/app/src/main/java/io/adrop/ads/example/model/Post.kt
package io.adrop.ads.example.model

import java.util.UUID
import kotlin.random.Random
import kotlin.random.nextInt

data class Post (
    val id: String = UUID.randomUUID().toString(),
    val icon: String = "https://storage.adrop.io/public/ic_openrhapsody.png",
    val title: String = "Adrop",
    val subTitle: String = "OpenRhapsody",
    val image: String = "https://storage.adrop.io/public/openrhapsody_title.png",
    val content: String = "Adrop Post Text",
    val like: Int = Random.nextInt(0..100),
    val comment: Int = Random.nextInt(0..100)
)
